boards/waveshare-esp32-s3-zero.json

```diff
@@ -0,0 +1,47 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32s3_out.ld",
+      "memory_type": "qio_qspi"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DBOARD_HAS_PSRAM",
+      "-DARDUINO_ESP32_S3_ZERO",
+      "-DARDUINO_USB_MODE=1",
+      "-DARDUINO_USB_CDC_ON_BOOT=1"
+    ],
```

Comment:
Aren't the `-DARDUINO_RUNNING_CORE=1` and `-DARDUINO_EVENT_RUNNING_CORE=1` macros needed here?